generated/nidaqmx/_task_modules/channels/ai_channel.py

```diff
@@ -2,6 +2,8 @@
 
 import ctypes
 import numpy
+import deprecation
+import __init__
```

Comment:
Did you test this? It doesn't work on my machine. 

`import __init__` imports a global module named `__init__`, not a submodule. I think it's loading `nidaqmx_examples/__init__.py`.
```
PS Z:\nidaqmx-python> poetry run python .\nidaqmx_examples\ai_voltage_sw_timed.py
Traceback (most recent call last):
  File ".\nidaqmx_examples\ai_voltage_sw_timed.py", line 2, in <module>
    import nidaqmx
  File "\\xidax-xyzzy\dev\nidaqmx-python\src\nidaqmx\__init__.py", line 3, in <module>
    from nidaqmx.task import Task
  File "\\xidax-xyzzy\dev\nidaqmx-python\src\nidaqmx\task.py", line 7, in <module>
    from nidaqmx._task_modules.channels.channel import Channel
  File "\\xidax-xyzzy\dev\nidaqmx-python\src\nidaqmx\_task_modules\channels\__init__.py", line 12, in <module>
    from nidaqmx._task_modules.channels.ai_channel import AIChannel
  File "\\xidax-xyzzy\dev\nidaqmx-python\src\nidaqmx\_task_modules\channels\ai_channel.py", line 36, in <module>
    __version__ = __init__.__version__
AttributeError: module '__init__' has no attribute '__version__'
```

If I use `from . import __init__`, it fails in a different way:

```
PS Z:\nidaqmx-python> poetry run python .\nidaqmx_examples\ai_voltage_sw_timed.py
Traceback (most recent call last):
  File ".\nidaqmx_examples\ai_voltage_sw_timed.py", line 2, in <module>
    import nidaqmx
  File "\\xidax-xyzzy\dev\nidaqmx-python\src\nidaqmx\__init__.py", line 3, in <module>
    from nidaqmx.task import Task
  File "\\xidax-xyzzy\dev\nidaqmx-python\src\nidaqmx\task.py", line 7, in <module>
    from nidaqmx._task_modules.channels.channel import Channel
  File "\\xidax-xyzzy\dev\nidaqmx-python\src\nidaqmx\_task_modules\channels\__init__.py", line 12, in <module>
    from nidaqmx._task_modules.channels.ai_channel import AIChannel
  File "\\xidax-xyzzy\dev\nidaqmx-python\src\nidaqmx\_task_modules\channels\ai_channel.py", line 36, in <module>
    __version__ = __init__.__version__
AttributeError: 'method-wrapper' object has no attribute '__version__'
```

https://stackoverflow.com/questions/50381749/python-3-how-do-i-import-init-py-from-a-module-in-its-package sort of explains what's going on here. It suggests a workaround, which seems to work:
```
from .. import channels

__version__ = channels.__version__
```

However, I think this comes very close to introducing a circular dependency, which seems brittle. (Consider why `import nidaqmx` in a submodule doesn't allow the submodule to use `nidaqmx.__version__`.)

Since we're only using `deprecated_in` right now, not `removed_in`, I think it would be simpler to remove `current_version=`.